torch/autograd/forward_ad.py

```diff
@@ -20,7 +20,8 @@
 
 
 def enter_dual_level():
-    r"""Function that can be used to enter a new forward grad level.
+    r"""Create a function to enter a new forward grad level.
```

Comment:
What does the function does? It doesn't "Create a function"

Reply:
Alright. I have updated it. Please review it.